nixos/modules/system/boot/systemd-user.nix

```diff
@@ -0,0 +1,172 @@
+{
+  config,
+  lib,
+  pkgs,
+  utils,
+  ...
+}:
+with utils;
+with systemdUtils.unitOptions;
+with lib; let
+  cfg = config.systemd.user;
+
+  systemd = config.systemd.package;
+
+  inherit
+    (systemdUtils.lib)
+    makeUnit
+    generateUnits
+    makeJobScript
+    unitConfig
+    serviceConfig
+    commonUnitText
+    targetToUnit
+    serviceToUnit
+    socketToUnit
+    timerToUnit
+    pathToUnit
+    ;
+
+  upstreamUserUnits =
+    [
+      "app.slice"
+      "background.slice"
+      "basic.target"
+      "bluetooth.target"
+      "default.target"
+      "exit.target"
+      "graphical-session-pre.target"
+      "graphical-session.target"
+      "paths.target"
+      "printer.target"
+      "session.slice"
+      "shutdown.target"
+      "smartcard.target"
+      "sockets.target"
+      "sound.target"
+      "systemd-exit.service"
+      "timers.target"
+      "xdg-desktop-autostart.target"
+    ]
+    ++ config.systemd.additionalUpstreamUserUnits;
```

Comment:
That isn't the formatting that was used before. Lets keep that and only split things

https://github.com/NixOS/nixpkgs/pull/164016/files#diff-5ea4693beb4bdd8f7efcdd3204ceea67c86f59b3d1c28cde063ddf03e1d20fbfL188-L210

Reply:
I redid the commits to align them with the original formatting again :+1: